### PR TITLE
Refactor the imports - Refs #34

### PR DESCRIPTION
**Pull request by hiromipaw**
Imports resource directly in cliquet and change BaseResource to Resource. As per issue #34

### CODE REVIEW

**Comment by hiromipaw** [comment]
I have some dependency to fix. Sorry for being late with this.

File: docs/reference/resource.rst [hunk 1]
```diff
@@ -15,12 +15,12 @@ Full example
...
     import colander
 
-    from cliquet import resource
+    import cliquet
     from cliquet import schema
     from cliquet import utils
 
 
-    class BookmarkSchema(resource.ResourceSchema):
+    class BookmarkSchema(cliquet.ResourceSchema):
```
Comment (leplatrem) [inline docs/reference/resource.rst]: I suppose we could even rename ``cliquet.ResourceSchema`` to ``cliquet.Schema`` ?

File: cliquet/__init__.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 """
 import warnings
 import pkg_resources
+import resource
```
Comment (leplatrem) [inline cliquet/__init__.py]: this bloc of imports groups the *stdlib* imports.

It should be ``from cliquet import resource`` along with others below like utils, etc.

File: cliquet/resource.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
                                     HTTPNotFound, HTTPConflict)
 import six
 
-from cliquet import logger
+from cliquet.logger import *
```
Comment (leplatrem) [inline cliquet/resource.py]: Importing ``*`` is not recommanded. Any good reason to have it?

**Comment by leplatrem** [comment]
Please don't say you're late :)
We're still breaking things anyway :)

Thanks a lot for your contribs!!

File: docs/reference/resource.rst (hunk 1 above)
Comment (ametaireau) [inline docs/reference/resource.rst]: +1

Comment (hiromipaw) [inline docs/reference/resource.rst]: :) allrighty

File: cliquet/resource.py (hunk 3 above)
Comment (hiromipaw) [inline cliquet/resource.py]: Nope was doing a test to understand why I was getting a couldn't find logger kind of error... :)

**Comment by hiromipaw** [comment]
Hey I have the same tests failing also on master. Is something with my local setup or am I forgetting something basic... ?

File: cliquet/tests/resource/test_views_cors.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import mock
-
+import cliquet
```
Comment (Natim) [inline cliquet/tests/resource/test_views_cors.py]: Is it necessary?

File: cliquet/__init__.py [hunk 5]
```diff
@@ -2,8 +2,8 @@
 """
 import warnings
 import pkg_resources
-
```
Comment (Natim) [inline cliquet/__init__.py]: Here we separate imports from the sdlib from imports from other libraries having ``import`` first and then ``from``

File: cliquet/errors.py [hunk 6]
```diff
@@ -1,6 +1,5 @@
 import six
 from pyramid import httpexceptions
-
```
Comment (Natim) [inline cliquet/errors.py]: ditto

File: cliquet/__init__.py [hunk 7]
```diff
@@ -13,7 +13,8 @@
 from cliquet import utils
 from cliquet.initialization import (  # NOQA
     initialize, initialize_cliquet, install_middlewares)
-
+from cliquet import resource
```
Comment (Natim) [inline cliquet/__init__.py]: Here it is not what we want.

We should probably define __all__ to define the exaustive list of what we want to expose in cliquet itself.

See https://github.com/django/django/blob/master/django/core/files/__init__.py#L3

**Comment by Natim** [comment]
refs https://github.com/nopressurelabs/cliquet/pull/1

**Comment by hiromipaw** [comment]
Thanks for the comments. I'll keep working on this. :)

**Pull request closed without merging**